config/default.schema.json

```diff
@@ -258,6 +264,11 @@
 				}
 			}
 		},
+		"NEXTCLOUD_DOMAIN": {
+			"type": "string",
+			"default": ".dbildungscloud.dev",
+			"description": "The Nextcloud domain"
+		},
```

Comment:
Why is this not added by an PR to the configuration at the dof_app_deploy repo?

Reply:
it was a first draft. just forgot to remove it again